vividus-plugin-web-app/src/test/java/org/vividus/bdd/steps/ui/web/WaitStepsTests.java

```diff
@@ -77,6 +83,7 @@
     @Mock private IBaseValidations baseValidations;
     @Mock private ISearchActions searchActions;
     @Mock private WebJavascriptActions javascriptActions;
+    @Spy private TimeoutConfigurer timeoutConfigurer;
```

Comment:
it's better to have a separate unit tests for `WaitSteps` and `TimeoutConfigurer`, in case of removal/refactoring of `WaitSteps`, the test coverage of `TimeoutConfigurer` will degraded as a side effect:
```suggestion
    @Mock private TimeoutConfigurer timeoutConfigurer;
```

Reply:
> TimeoutConfigurer

But TimeoutConfigurer already has its own tests
https://github.com/vividus-framework/vividus/blob/master/vividus-plugin-web-app/src/test/java/org/vividus/selenium/TimeoutConfigurerTests.java

Or it makes sense to add a test for the method "configurePageLoadTimeout(Duration duration, Timeouts timeouts)"?

Reply:
> it makes sense to add a test for the method "configurePageLoadTimeout(Duration duration, Timeouts timeouts)"
yes, exactly